src/popup/popup.js

```diff
@@ -10,6 +12,11 @@ function main() {
     document.getElementById("email-input-form").addEventListener("keypress", submitLoginOnKeyEnter);
     document.getElementById("password-input-form").addEventListener("keypress", submitLoginOnKeyEnter);
     document.getElementById("idp-input-form").addEventListener("keypress", submitLoginOnKeyEnter);
+    document.getElementById("domain-input-form").addEventListener("keypress", submitLoginOnKeyEnter);
```

Comment:
Why is this only one field? What if a user wants to add multiple domains? Is it expected that they all add it in a single regex?

Reply:
Having multiple input forms for different domain filters would make the popup too cluttered.
In the long term, I am planning to move all configurations to an [option page](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Options_pages) where we could also implement the use of multiple Client ID's and multiple domain filters per Client ID in a more structured manner.